test/ered_SUITE.erl

```diff
@@ -81,30 +99,39 @@ reset_cluster() ->
 %% Wait until cluster is consistent, i.e all nodes have the same single view
 %% of the slot map and all cluster nodes are included in the slot map.
 wait_for_consistent_cluster() ->
+    wait_for_consistent_cluster(?PORTS).
+
+wait_for_consistent_cluster(Ports) ->
     fun Loop(N) ->
-            SlotMaps = [fun(Port) ->
-                                {ok, Pid} = ered_client:start_link("127.0.0.1", Port, []),
-                                {ok, SlotMap} = ered_client:command(Pid, [<<"CLUSTER">>, <<"SLOTS">>]),
-                                ered_client:stop(Pid),
-                                SlotMap
-                        end(P) || P <- ?PORTS],
-            Consistent = case lists:usort(SlotMaps) of
-                             [SlotMap] ->
-                                 ?PORTS =:= [Port || {_Ip, Port} <- ered_lib:slotmap_all_nodes(SlotMap)];
-                             _NotAllIdentical ->
-                                 false
-                         end,
-            case Consistent of
-                true ->
+            case check_cluster_consistent(Ports) of
+                ok ->
                     true;
-                false when N > 0 ->
+                {error, _} when N > 0 ->
                     timer:sleep(500),
                     Loop(N-1);
-                false ->
+                {error, SlotMaps} ->
                     error({timeout_consistent_cluster, SlotMaps})
             end
     end(20).
 
+check_cluster_consistent(Ports) ->
```

Comment:
I should probably rename this to `check_consisten_cluster` to match the name `wait_for_consistent_cluster`.